docs/explanation/index.rst

```diff
@@ -16,6 +16,7 @@ and their implementation.
     parts-lifecycle
     extensions
     components
+    interfaces
```

Comment:
I think it should come after _Parts lifecycle_.

Reply:
https://github.com/canonical/snapcraft/pull/5464/commits/aef48b8d2374797eea498351655725de6f646ff0